compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/lazy/staggeredgrid/LazyStaggeredGridMeasureResult.kt

```diff
@@ -147,4 +153,5 @@ internal object EmptyLazyStaggeredGridLayoutInfo : LazyStaggeredGridLayoutInfo {
     override val viewportEndOffset: Int = 0
     override val beforeContentPadding: Int = 0
     override val afterContentPadding: Int = 0
+    override val mainAxisItemSpacing: Int = 0
```

Comment:
Better to wait for https://android-review.googlesource.com/c/platform/frameworks/support/+/2397134 to merge, and then cherry-pick it straight into `jb-main`

Reply:
Done. The cherry-pick is part of this PR, so you can review it too.
Once approved, I will `merge --ff-only` this branch into `jb-main`, to avoid squashing the two commits.

Reply:
Let's push the cherry-pick straight into jb-main.

The alternative process is as you described. It is better because we review everything what adds to the main branch, but worse because:
1. we have to enable merging of PR's. If we do that, someone can use it, and don't bother about commits in the PR. It will spoil the git history.
2. the review of a cherry-pick and a fix together is harder than just fixes.
3. the 2. can be fixed by squashing the fix into a single commit, but it requires force-push, that is also not good for review.

Reply:
On the other hand, we also lose ability to run CI on the cherry-pick.

And one more issue. When it will be merged, the code is become common. And when users tried to use it, they encounter noSuchMethod on Android. I don't know good way to solve it. We have 2 options:
1. wait for the Jetpack release, and point to it in our sources. But if it will be 1.5, we can't merge it right now
2. make public API internal, fix conflicts later (we should add comments near internal to describe how to solve conflicts)

Reply:
I'll adjust the cherry-picked commit to make the new properties `internal` for now.